Gemfile

```diff
@@ -4,13 +4,21 @@ case ENV.fetch('DATABASE_ADAPTER', nil)
 when 'mongoid' then
   gem 'kaminari-mongoid'
   gem 'mongoid', ENV['MONGOID_VERSION'] || '~> 7.3.0'
-  gem 'mongoid-scroll'
+  gem 'mongoid-scroll', '~> 1.0.1'
```

Comment:
This one had a breaking change in 2.0 that's easy to fix. It's something like this (from https://github.com/dblock/slack-strava/commit/88af16b76703c4a4a2919468b9c2dbd2bc42a46f#diff-633cee3ca2d39648d625e12b921138aa0a932ebe9375ba53102cbabc339e5afe).
 
![Screenshot 2024-12-01 at 7 03 38 PM](https://github.com/user-attachments/assets/3f759aee-d52d-4551-85ef-6da855990862)